src/api/article/models.js

```diff
@@ -0,0 +1,53 @@
+import mongoose, { Schema } from 'mongoose';
+import User from 'api/user';
+
+const ArticleTypeSchema = new Schema({
+  type: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+});
+
+export const ArticleType = mongoose.model('ArticleType', ArticleTypeSchema);
+
+const ArticleSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  subtitle: {
+    type: String,
+    required: true,
+  },
+  video: String,
+  image: String,
+  text: Schema.Types.Mixed,
+  author: {
+    type: Schema.Types.ObjectId,
+    ref: User,
+  },
+  tags: [String],
+  collectionId: {
+    type: Schema.Types.ObjectId,
+    // TODO: add reference to ArticleCollection
+  },
+  type: {
+    type: Schema.Types.ObjectId,
+    required: true,
+    ref: ArticleType,
+  },
+  slug: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now(),
```

Comment:
**BUG!!!** you always will get the same value of one call `Date.now()`, you must give a `function` to `default` and in this case it'll be called on create